zip_file/versions.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = "~> 0.12.25"
+
+  required_providers {
+    aws = {
+      #source  = "hashicorp/aws"
```

Comment:
Is this required?

Reply:
Yes and no.  From terraform 13 onwards the commented out line is required to pass tflinting, but it's not compatible with terraform 12, so it's commented out for now.

Reply:
So should we create a version compatible with tf12 without the line, and then a further update for tf13 with the line in?

Reply:
That is the option.  Go on then, I've pushed it without the comment ;-)